transforms/universal/ededup/python/README.md

```diff
@@ -23,12 +30,17 @@ The following command line arguments are available in addition to
 the options provided by 
 the [python launcher](../../../../data-processing-lib/doc/python-launcher-options.md).
 ```
- --ededup_doc_column EDEDUP_DOC_COLUMN
-                        name of the column containing document 
+  --ededup_doc_column EDEDUP_DOC_COLUMN
+                        key for accessing data
```

Comment:
I think the previous definition was clearer.

Reply:
fixed